lms/resources/_js_config/__init__.py

```diff
@@ -510,3 +503,43 @@ def _sync_api(self):
             }
 
         return sync_api_config
+
+    def _blackboard_sync_api(self):
+        req = self._request
+        return {
+            "authUrl": req.route_url("blackboard_api.oauth.authorize"),
+            "path": req.route_path("blackboard_api.sync"),
+            "data": {
+                "lms": {
+                    "tool_consumer_instance_guid": req.params[
+                        "tool_consumer_instance_guid"
+                    ],
+                },
+                "course": {
+                    "context_id": req.params["context_id"],
+                    "resource_link_id": req.params["resource_link_id"],
+                },
+                "group_info": {
+                    key: value
+                    for key, value in req.params.items()
+                    if key in GroupInfo.columns()
+                },
+            },
+        }
+
+    def _sync_api(self):
+        if self._context.is_canvas and (
+            self._context.canvas_sections_enabled
+            or self._context.canvas_is_group_launch
+        ):
+            return self._canvas_sync_api()
+
+        if (
+            self._application_instance().product
+            == ApplicationInstance.Product.BLACKBOARD
+            and self._context.is_blackboard_group_launch
```

Comment:
I noticed a naming inconsistency between `is_blackboard_group_launch` and `canvas_is_group_launch`. It's not from this PR though